consul/data_source_consul_service.go

```diff
@@ -111,6 +112,10 @@ func dataSourceConsulService() *schema.Resource {
 							Computed: true,
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
+						catalogServiceServiceMeta: {
+							Type:     schema.TypeMap,
+							Computed: true,
```

Comment:
Wouldn't the `Elem` key be required here?

```go
		Elem: &schema.Schema{
			Type: schema.TypeString,
		},
```

Reply:
Thanks, I did not know that :)